panic: runtime error: slice bounds out of range [:226] with capacity 128

After upgrading to version 1.12.0 and after several successful GET queries, the container crashes with this error:

## Log

```
{"action":"lsm_init_disk_segment_build_bloom_filter_secondary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 151.178128ms\n","path":"/var/lib/weaviate/***_***_v1_VFrq4mcLtG0m_lsm/objects/segment-1645771599695395300.db","secondary_index_position":0,"shard":"VFrq4mcLtG0m","time":"2022-04-06T11:06:37Z","took":151178128}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 231.699696ms\n","path":"/var/lib/weaviate/***_***_v1_VFrq4mcLtG0m_lsm/objects/segment-1645771599695395300.db","shard":"VFrq4mcLtG0m","time":"2022-04-06T11:06:37Z","took":231699696}
{"action":"lsm_init_disk_segment_build_bloom_filter_secondary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 17.599317ms\n","path":"/var/lib/weaviate/***_***_v1_VFrq4mcLtG0m_lsm/objects/segment-1647065218321394711.db","secondary_index_position":0,"shard":"VFrq4mcLtG0m","time":"2022-04-06T11:06:46Z","took":17599317}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 15.928598ms\n","path":"/var/lib/weaviate/***_***_v1_VFrq4mcLtG0m_lsm/objects/segment-1647065218321394711.db","shard":"VFrq4mcLtG0m","time":"2022-04-06T11:06:46Z","took":15928598}
{"action":"hnsw_vector_cache_prefill_level","count":383665,"hnsw_level":0,"index_id":"***_***_v1_oV6sUJRrdRQk","level":"debug","msg":"prefilled level in vector cache","time":"2022-04-06T11:06:47Z","took":10614323471}
{"action":"hnsw_vector_cache_prefill","count":450000,"index_id":"***_***_v1_oV6sUJRrdRQk","level":"info","limit":2000000,"msg":"prefilled vector cache","time":"2022-04-06T11:06:47Z","took":10874590650}
{"action":"lsm_init_disk_segment_build_bloom_filter_secondary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 5.039284ms\n","path":"/var/lib/weaviate/***_***_v1_VFrq4mcLtG0m_lsm/objects/segment-1647323966635288876.db","secondary_index_position":0,"shard":"VFrq4mcLtG0m","time":"2022-04-06T11:06:47Z","took":5039284}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 4.169362ms\n","path":"/var/lib/weaviate/***_***_v1_VFrq4mcLtG0m_lsm/objects/segment-1647323966635288876.db","shard":"VFrq4mcLtG0m","time":"2022-04-06T11:06:47Z","took":4169362}
{"action":"lsm_init_disk_segment_build_bloom_filter_secondary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 21.818µs\n","path":"/var/lib/weaviate/***_***_v1_VFrq4mcLtG0m_lsm/objects/segment-1647324341156587467.db","secondary_index_position":0,"shard":"VFrq4mcLtG0m","time":"2022-04-06T11:06:48Z","took":21818}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 11.78µs\n","path":"/var/lib/weaviate/***_***_v1_VFrq4mcLtG0m_lsm/objects/segment-1647324341156587467.db","shard":"VFrq4mcLtG0m","time":"2022-04-06T11:06:48Z","took":11780}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 689.402µs\n","path":"/var/lib/weaviate/***_***_v1_VFrq4mcLtG0m_lsm/property_class_id/segment-1647065590169697628.db","shard":"VFrq4mcLtG0m","time":"2022-04-06T11:06:48Z","took":689402}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 5.985µs\n","path":"/var/lib/weaviate/***_***_v1_VFrq4mcLtG0m_lsm/property_class_id/segment-1647259451560870008.db","shard":"VFrq4mcLtG0m","time":"2022-04-06T11:06:48Z","took":5985}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 2.242µs\n","path":"/var/lib/weaviate/***_***_v1_VFrq4mcLtG0m_lsm/property_class_id/segment-1647324341158636755.db","shard":"VFrq4mcLtG0m","time":"2022-04-06T11:06:48Z","took":2242}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 3.728µs\n","path":"/var/lib/weaviate/***_***_v1_VFrq4mcLtG0m_lsm/hash_property_class_id/segment-1647065590175686403.db","shard":"VFrq4mcLtG0m","time":"2022-04-06T11:06:48Z","took":3728}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 20.436µs\n","path":"/var/lib/weaviate/***_***_v1_VFrq4mcLtG0m_lsm/hash_property_class_id/segment-1647259451561499169.db","shard":"VFrq4mcLtG0m","time":"2022-04-06T11:06:48Z","took":20436}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 3.328µs\n","path":"/var/lib/weaviate/***_***_v1_VFrq4mcLtG0m_lsm/hash_property_class_id/segment-1647324341160666369.db","shard":"VFrq4mcLtG0m","time":"2022-04-06T11:06:48Z","took":3328}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 120.256269ms\n","path":"/var/lib/weaviate/***_***_v1_VFrq4mcLtG0m_lsm/property_***/segment-1647065590318913885.db","shard":"VFrq4mcLtG0m","time":"2022-04-06T11:06:48Z","took":120256269}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 1.572676ms\n","path":"/var/lib/weaviate/***_***_v1_VFrq4mcLtG0m_lsm/property_***/segment-1647259451672328077.db","shard":"VFrq4mcLtG0m","time":"2022-04-06T11:06:48Z","took":1572676}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 984.419µs\n","path":"/var/lib/weaviate/***_***_v1_VFrq4mcLtG0m_lsm/property_***/segment-1647324341290095454.db","shard":"VFrq4mcLtG0m","time":"2022-04-06T11:06:48Z","took":984419}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 117.962954ms\n","path":"/var/lib/weaviate/***_***_v1_VFrq4mcLtG0m_lsm/hash_property_***/segment-1647065590475689095.db","shard":"VFrq4mcLtG0m","time":"2022-04-06T11:06:48Z","took":117962954}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 1.350155ms\n","path":"/var/lib/weaviate/***_***_v1_VFrq4mcLtG0m_lsm/hash_property_***/segment-1647259451791977670.db","shard":"VFrq4mcLtG0m","time":"2022-04-06T11:06:48Z","took":1350155}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 680.167µs\n","path":"/var/lib/weaviate/***_***_v1_VFrq4mcLtG0m_lsm/hash_property_***/segment-1647324341425125411.db","shard":"VFrq4mcLtG0m","time":"2022-04-06T11:06:48Z","took":680167}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 119.459894ms\n","path":"/var/lib/weaviate/***_***_v1_VFrq4mcLtG0m_lsm/property_***/segment-1647065590623482868.db","shard":"VFrq4mcLtG0m","time":"2022-04-06T11:06:48Z","took":119459894}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 1.441782ms\n","path":"/var/lib/weaviate/***_***_v1_VFrq4mcLtG0m_lsm/property_***/segment-1647259451897954264.db","shard":"VFrq4mcLtG0m","time":"2022-04-06T11:06:48Z","took":1441782}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 1.04269ms\n","path":"/var/lib/weaviate/***_***_v1_VFrq4mcLtG0m_lsm/property_***/segment-1647324341560303808.db","shard":"VFrq4mcLtG0m","time":"2022-04-06T11:06:48Z","took":1042690}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 120.078554ms\n","path":"/var/lib/weaviate/***_***_v1_VFrq4mcLtG0m_lsm/hash_property_***/segment-1647065590752731945.db","shard":"VFrq4mcLtG0m","time":"2022-04-06T11:06:48Z","took":120078554}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 1.291064ms\n","path":"/var/lib/weaviate/***_***_v1_VFrq4mcLtG0m_lsm/hash_property_***/segment-1647259452018129475.db","shard":"VFrq4mcLtG0m","time":"2022-04-06T11:06:48Z","took":1291064}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 677.674µs\n","path":"/var/lib/weaviate/***_***_v1_VFrq4mcLtG0m_lsm/hash_property_***/segment-1647324341686251861.db","shard":"VFrq4mcLtG0m","time":"2022-04-06T11:06:48Z","took":677674}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 881.219µs\n","path":"/var/lib/weaviate/***_***_v1_VFrq4mcLtG0m_lsm/property_***/segment-1647065590770665991.db","shard":"VFrq4mcLtG0m","time":"2022-04-06T11:06:48Z","took":881219}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 89.501µs\n","path":"/var/lib/weaviate/***_***_v1_VFrq4mcLtG0m_lsm/property_***/segment-1647259452019107941.db","shard":"VFrq4mcLtG0m","time":"2022-04-06T11:06:48Z","took":89501}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 50.47µs\n","path":"/var/lib/weaviate/***_***_v1_VFrq4mcLtG0m_lsm/property_***/segment-1647324341713885678.db","shard":"VFrq4mcLtG0m","time":"2022-04-06T11:06:48Z","took":50470}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 143.509µs\n","path":"/var/lib/weaviate/***_***_v1_VFrq4mcLtG0m_lsm/hash_property_***/segment-1647065590777451131.db","shard":"VFrq4mcLtG0m","time":"2022-04-06T11:06:48Z","took":143509}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 70.927µs\n","path":"/var/lib/weaviate/***_***_v1_VFrq4mcLtG0m_lsm/hash_property_***/segment-1647259452041702483.db","shard":"VFrq4mcLtG0m","time":"2022-04-06T11:06:48Z","took":70927}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 49.392µs\n","path":"/var/lib/weaviate/***_***_v1_VFrq4mcLtG0m_lsm/hash_property_***/segment-1647324341716536936.db","shard":"VFrq4mcLtG0m","time":"2022-04-06T11:06:48Z","took":49392}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 883.579µs\n","path":"/var/lib/weaviate/***_***_v1_VFrq4mcLtG0m_lsm/property_***/segment-1647065590783513106.db","shard":"VFrq4mcLtG0m","time":"2022-04-06T11:06:48Z","took":883579}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 20.754µs\n","path":"/var/lib/weaviate/***_***_v1_VFrq4mcLtG0m_lsm/property_***/segment-1647259452042408063.db","shard":"VFrq4mcLtG0m","time":"2022-04-06T11:06:48Z","took":20754}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 16.716µs\n","path":"/var/lib/weaviate/***_***_v1_VFrq4mcLtG0m_lsm/property_***/segment-1647324341718638185.db","shard":"VFrq4mcLtG0m","time":"2022-04-06T11:06:48Z","took":16716}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 33.596µs\n","path":"/var/lib/weaviate/***_***_v1_VFrq4mcLtG0m_lsm/hash_property_***/segment-1647065590789275330.db","shard":"VFrq4mcLtG0m","time":"2022-04-06T11:06:48Z","took":33596}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 24.988µs\n","path":"/var/lib/weaviate/***_***_v1_VFrq4mcLtG0m_lsm/hash_property_***/segment-1647259452043053662.db","shard":"VFrq4mcLtG0m","time":"2022-04-06T11:06:48Z","took":24988}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 16.886µs\n","path":"/var/lib/weaviate/***_***_v1_VFrq4mcLtG0m_lsm/hash_property_***/segment-1647324341721064275.db","shard":"VFrq4mcLtG0m","time":"2022-04-06T11:06:48Z","took":16886}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 601.261µs\n","path":"/var/lib/weaviate/***_***_v1_VFrq4mcLtG0m_lsm/property_***/segment-1647065590793773013.db","shard":"VFrq4mcLtG0m","time":"2022-04-06T11:06:48Z","took":601261}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 2.07µs\n","path":"/var/lib/weaviate/***_***_v1_VFrq4mcLtG0m_lsm/property_***/segment-1647259452043609671.db","shard":"VFrq4mcLtG0m","time":"2022-04-06T11:06:48Z","took":2070}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 1.784µs\n","path":"/var/lib/weaviate/***_***_v1_VFrq4mcLtG0m_lsm/property_***/segment-1647324341723414480.db","shard":"VFrq4mcLtG0m","time":"2022-04-06T11:06:48Z","took":1784}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 2.484µs\n","path":"/var/lib/weaviate/***_***_v1_VFrq4mcLtG0m_lsm/hash_property_***/segment-1647065590798706165.db","shard":"VFrq4mcLtG0m","time":"2022-04-06T11:06:48Z","took":2484}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 11.45µs\n","path":"/var/lib/weaviate/***_***_v1_VFrq4mcLtG0m_lsm/hash_property_***/segment-1647259452044209456.db","shard":"VFrq4mcLtG0m","time":"2022-04-06T11:06:48Z","took":11450}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 6.333µs\n","path":"/var/lib/weaviate/***_***_v1_VFrq4mcLtG0m_lsm/hash_property_***/segment-1647324341725566412.db","shard":"VFrq4mcLtG0m","time":"2022-04-06T11:06:48Z","took":6333}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 155.175189ms\n","path":"/var/lib/weaviate/***_***_v1_VFrq4mcLtG0m_lsm/property__id/segment-1647065590954448998.db","shard":"VFrq4mcLtG0m","time":"2022-04-06T11:06:48Z","took":155175189}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 2.445471ms\n","path":"/var/lib/weaviate/***_***_v1_VFrq4mcLtG0m_lsm/property__id/segment-1647259452168426193.db","shard":"VFrq4mcLtG0m","time":"2022-04-06T11:06:48Z","took":2445471}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 1.827367ms\n","path":"/var/lib/weaviate/***_***_v1_VFrq4mcLtG0m_lsm/property__id/segment-1647324341857844933.db","shard":"VFrq4mcLtG0m","time":"2022-04-06T11:06:48Z","took":1827367}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 142.274406ms\n","path":"/var/lib/weaviate/***_***_v1_VFrq4mcLtG0m_lsm/hash_property__id/segment-1647065591101980309.db","shard":"VFrq4mcLtG0m","time":"2022-04-06T11:06:49Z","took":142274406}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 2.453538ms\n","path":"/var/lib/weaviate/***_***_v1_VFrq4mcLtG0m_lsm/hash_property__id/segment-1647259452289133786.db","shard":"VFrq4mcLtG0m","time":"2022-04-06T11:06:49Z","took":2453538}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 1.404494ms\n","path":"/var/lib/weaviate/***_***_v1_VFrq4mcLtG0m_lsm/hash_property__id/segment-1647324341991736901.db","shard":"VFrq4mcLtG0m","time":"2022-04-06T11:06:49Z","took":1404494}
{"action":"lsm_init_disk_segment_build_bloom_filter_secondary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 4.512µs\n","path":"/var/lib/weaviate/***_***_v1_zOnLKnrAcjHE_lsm/objects/segment-1645188247966319301.db","secondary_index_position":0,"shard":"zOnLKnrAcjHE","time":"2022-04-06T11:06:49Z","took":4512}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 1.768µs\n","path":"/var/lib/weaviate/***_***_v1_zOnLKnrAcjHE_lsm/objects/segment-1645188247966319301.db","shard":"zOnLKnrAcjHE","time":"2022-04-06T11:06:49Z","took":1768}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 4.665µs\n","path":"/var/lib/weaviate/***_***_v1_zOnLKnrAcjHE_lsm/property_class_id/segment-1645188247966370725.db","shard":"zOnLKnrAcjHE","time":"2022-04-06T11:06:49Z","took":4665}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 4.575µs\n","path":"/var/lib/weaviate/***_***_v1_zOnLKnrAcjHE_lsm/hash_property_class_id/segment-1645188247966401341.db","shard":"zOnLKnrAcjHE","time":"2022-04-06T11:06:49Z","took":4575}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 4.403µs\n","path":"/var/lib/weaviate/***_***_v1_zOnLKnrAcjHE_lsm/property_***/segment-1645188247966432301.db","shard":"zOnLKnrAcjHE","time":"2022-04-06T11:06:49Z","took":4403}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 4.699µs\n","path":"/var/lib/weaviate/***_***_v1_zOnLKnrAcjHE_lsm/hash_property_***/segment-1645188247966465772.db","shard":"zOnLKnrAcjHE","time":"2022-04-06T11:06:49Z","took":4699}
{"action":"hnsw_vector_cache_prefill_level","count":1,"hnsw_level":3,"index_id":"***_***_v1_VFrq4mcLtG0m","level":"debug","msg":"prefilled level in vector cache","time":"2022-04-06T11:06:49Z","took":14134444}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 3.23µs\n","path":"/var/lib/weaviate/***_***_v1_zOnLKnrAcjHE_lsm/property_***/segment-1645188247966497739.db","shard":"zOnLKnrAcjHE","time":"2022-04-06T11:06:49Z","took":3230}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 3.53µs\n","path":"/var/lib/weaviate/***_***_v1_zOnLKnrAcjHE_lsm/hash_property_***/segment-1645188247966527451.db","shard":"zOnLKnrAcjHE","time":"2022-04-06T11:06:49Z","took":3530}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 2.672µs\n","path":"/var/lib/weaviate/***_***_v1_zOnLKnrAcjHE_lsm/property_***/segment-1645188247966555401.db","shard":"zOnLKnrAcjHE","time":"2022-04-06T11:06:49Z","took":2672}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 4.547µs\n","path":"/var/lib/weaviate/***_***_v1_zOnLKnrAcjHE_lsm/hash_property_***/segment-1645188247966592241.db","shard":"zOnLKnrAcjHE","time":"2022-04-06T11:06:49Z","took":4547}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 3.025µs\n","path":"/var/lib/weaviate/***_***_v1_zOnLKnrAcjHE_lsm/property_***/segment-1645188247966625728.db","shard":"zOnLKnrAcjHE","time":"2022-04-06T11:06:49Z","took":3025}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 3.94µs\n","path":"/var/lib/weaviate/***_***_v1_zOnLKnrAcjHE_lsm/hash_property_***/segment-1645188247966656392.db","shard":"zOnLKnrAcjHE","time":"2022-04-06T11:06:49Z","took":3940}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 2.592µs\n","path":"/var/lib/weaviate/***_***_v1_zOnLKnrAcjHE_lsm/property_***/segment-1645188247966708411.db","shard":"zOnLKnrAcjHE","time":"2022-04-06T11:06:49Z","took":2592}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 3.12µs\n","path":"/var/lib/weaviate/***_***_v1_zOnLKnrAcjHE_lsm/hash_property_***/segment-1645188247966736686.db","shard":"zOnLKnrAcjHE","time":"2022-04-06T11:06:49Z","took":3120}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 4.104µs\n","path":"/var/lib/weaviate/***_***_v1_zOnLKnrAcjHE_lsm/property__id/segment-1645188247966887036.db","shard":"zOnLKnrAcjHE","time":"2022-04-06T11:06:49Z","took":4104}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 3.176µs\n","path":"/var/lib/weaviate/***_***_v1_zOnLKnrAcjHE_lsm/hash_property__id/segment-1645188247966917742.db","shard":"zOnLKnrAcjHE","time":"2022-04-06T11:06:49Z","took":3176}
{"action":"hnsw_vector_cache_prefill_level","count":62,"hnsw_level":2,"index_id":"***_***_v1_VFrq4mcLtG0m","level":"debug","msg":"prefilled level in vector cache","time":"2022-04-06T11:06:49Z","took":15617589}
{"action":"hnsw_vector_cache_prefill_level","count":1,"hnsw_level":0,"index_id":"***_***_v1_zOnLKnrAcjHE","level":"debug","msg":"prefilled level in vector cache","time":"2022-04-06T11:06:49Z","took":912156}
{"action":"hnsw_vector_cache_prefill","count":50000,"index_id":"***_***_v1_zOnLKnrAcjHE","level":"info","limit":2000000,"msg":"prefilled vector cache","time":"2022-04-06T11:06:49Z","took":992287}
{"action":"lsm_init_disk_segment_build_bloom_filter_secondary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 4.166µs\n","path":"/var/lib/weaviate/***_***_v1_MbWDCEVTP0Uw_lsm/objects/segment-1645692926113595696.db","secondary_index_position":0,"shard":"MbWDCEVTP0Uw","time":"2022-04-06T11:06:49Z","took":4166}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 1.834µs\n","path":"/var/lib/weaviate/***_***_v1_MbWDCEVTP0Uw_lsm/objects/segment-1645692926113595696.db","shard":"MbWDCEVTP0Uw","time":"2022-04-06T11:06:49Z","took":1834}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 13.021µs\n","path":"/var/lib/weaviate/***_***_v1_MbWDCEVTP0Uw_lsm/property_class_id/segment-1645692926113659805.db","shard":"MbWDCEVTP0Uw","time":"2022-04-06T11:06:49Z","took":13021}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 4.438µs\n","path":"/var/lib/weaviate/***_***_v1_MbWDCEVTP0Uw_lsm/hash_property_class_id/segment-1645692926113704646.db","shard":"MbWDCEVTP0Uw","time":"2022-04-06T11:06:49Z","took":4438}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 4.167µs\n","path":"/var/lib/weaviate/***_***_v1_MbWDCEVTP0Uw_lsm/property_***/segment-1645692926113734742.db","shard":"MbWDCEVTP0Uw","time":"2022-04-06T11:06:49Z","took":4167}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 3.265µs\n","path":"/var/lib/weaviate/***_***_v1_MbWDCEVTP0Uw_lsm/hash_property_***/segment-1645692926113793377.db","shard":"MbWDCEVTP0Uw","time":"2022-04-06T11:06:49Z","took":3265}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 3.338µs\n","path":"/var/lib/weaviate/***_***_v1_MbWDCEVTP0Uw_lsm/property_***/segment-1645692926113846365.db","shard":"MbWDCEVTP0Uw","time":"2022-04-06T11:06:49Z","took":3338}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 2.885µs\n","path":"/var/lib/weaviate/***_***_v1_MbWDCEVTP0Uw_lsm/hash_property_***/segment-1645692926113892475.db","shard":"MbWDCEVTP0Uw","time":"2022-04-06T11:06:49Z","took":2885}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 2.711µs\n","path":"/var/lib/weaviate/***_***_v1_MbWDCEVTP0Uw_lsm/property_***/segment-1645692926113921927.db","shard":"MbWDCEVTP0Uw","time":"2022-04-06T11:06:49Z","took":2711}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 3.03µs\n","path":"/var/lib/weaviate/***_***_v1_MbWDCEVTP0Uw_lsm/hash_property_***/segment-1645692926113960892.db","shard":"MbWDCEVTP0Uw","time":"2022-04-06T11:06:49Z","took":3030}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 3.256µs\n","path":"/var/lib/weaviate/***_***_v1_MbWDCEVTP0Uw_lsm/property_***/segment-1645692926114009341.db","shard":"MbWDCEVTP0Uw","time":"2022-04-06T11:06:49Z","took":3256}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 3.163µs\n","path":"/var/lib/weaviate/***_***_v1_MbWDCEVTP0Uw_lsm/hash_property_***/segment-1645692926114057175.db","shard":"MbWDCEVTP0Uw","time":"2022-04-06T11:06:49Z","took":3163}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 3.676µs\n","path":"/var/lib/weaviate/***_***_v1_MbWDCEVTP0Uw_lsm/property_***/segment-1645692926114108415.db","shard":"MbWDCEVTP0Uw","time":"2022-04-06T11:06:49Z","took":3676}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 2.677µs\n","path":"/var/lib/weaviate/***_***_v1_MbWDCEVTP0Uw_lsm/hash_property_***/segment-1645692926114155995.db","shard":"MbWDCEVTP0Uw","time":"2022-04-06T11:06:49Z","took":2677}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 3.105µs\n","path":"/var/lib/weaviate/***_***_v1_MbWDCEVTP0Uw_lsm/property_***/segment-1645692926114303748.db","shard":"MbWDCEVTP0Uw","time":"2022-04-06T11:06:49Z","took":3105}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 2.967µs\n","path":"/var/lib/weaviate/***_***_v1_MbWDCEVTP0Uw_lsm/hash_property_***/segment-1645692926114341501.db","shard":"MbWDCEVTP0Uw","time":"2022-04-06T11:06:49Z","took":2967}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 3.98µs\n","path":"/var/lib/weaviate/***_***_v1_MbWDCEVTP0Uw_lsm/property__id/segment-1645692926114373102.db","shard":"MbWDCEVTP0Uw","time":"2022-04-06T11:06:49Z","took":3980}
{"action":"lsm_init_disk_segment_build_bloom_filter_primary","class":"***_***_v1","index":"***_***_v1","level":"debug","msg":"building bloom filter took 3.358µs\n","path":"/var/lib/weaviate/***_***_v1_MbWDCEVTP0Uw_lsm/hash_property__id/segment-1645692926114418043.db","shard":"MbWDCEVTP0Uw","time":"2022-04-06T11:06:49Z","took":3358}
{"action":"hnsw_vector_cache_prefill_level","count":1,"hnsw_level":0,"index_id":"***_***_v1_MbWDCEVTP0Uw","level":"debug","msg":"prefilled level in vector cache","time":"2022-04-06T11:06:49Z","took":967623}
{"action":"hnsw_vector_cache_prefill","count":50000,"index_id":"***_***_v1_MbWDCEVTP0Uw","level":"info","limit":2000000,"msg":"prefilled vector cache","time":"2022-04-06T11:06:49Z","took":1040377}
{"action":"hnsw_vector_cache_prefill_level","count":3533,"hnsw_level":1,"index_id":"***_***_v1_VFrq4mcLtG0m","level":"debug","msg":"prefilled level in vector cache","time":"2022-04-06T11:06:49Z","took":71957469}
{"action":"startup","level":"debug","msg":"start initializing modules","time":"2022-04-06T11:06:49Z"}
{"action":"startup","level":"debug","msg":"finished initializing modules","time":"2022-04-06T11:06:49Z"}
{"action":"graphql_rebuild","level":"debug","msg":"rebuilding the graphql schema","schema":{"Objects":{"classes":[{"class":"***_***_v1","description":"*** cz.","invertedIndexConfig":{"cleanupIntervalSeconds":60},"properties":[{"dataType":["int"],"name":"class_id"},{"dataType":["int"],"name":"***"},{"dataType":["int"],"name":"***"},{"dataType":["int"],"name":"***"},{"dataType":["int"],"name":"***"},{"dataType":["int"],"name":"***"},{"dataType":["int"],"name":"***"},{"dataType":["string"],"name":"***","tokenization":"word"}],"shardingConfig":{"virtualPerPhysical":128,"desiredCount":1,"actualCount":1,"desiredVirtualCount":128,"actualVirtualCount":128,"key":"_id","strategy":"hash","function":"murmur3"},"vectorIndexConfig":{"skip":false,"cleanupIntervalSeconds":300,"maxConnections":64,"efConstruction":128,"ef":-1,"dynamicEfMin":100,"dynamicEfMax":500,"dynamicEfFactor":8,"vectorCacheMaxObjects":2000000,"flatSearchCutoff":40000},"vectorIndexType":"hnsw","vectorizer":"none"},{"class":"***_proj3_v1","description":"*** sk.","invertedIndexConfig":{"cleanupIntervalSeconds":60},"properties":[{"dataType":["int"],"name":"class_id"},{"dataType":["int"],"name":"***"},{"dataType":["int"],"name":"***"},{"dataType":["int"],"name":"***"},{"dataType":["int"],"name":"***"},{"dataType":["int"],"name":"***"},{"dataType":["int"],"name":"***"},{"dataType":["string"],"name":"***","tokenization":"word"}],"shardingConfig":{"virtualPerPhysical":128,"desiredCount":1,"actualCount":1,"desiredVirtualCount":128,"actualVirtualCount":128,"key":"_id","strategy":"hash","function":"murmur3"},"vectorIndexConfig":{"skip":false,"cleanupIntervalSeconds":300,"maxConnections":64,"efConstruction":128,"ef":-1,"dynamicEfMin":100,"dynamicEfMax":500,"dynamicEfFactor":8,"vectorCacheMaxObjects":2000000,"flatSearchCutoff":40000},"vectorIndexType":"hnsw","vectorizer":"none"},{"class":"***_***_v1","description":"*** ro.","invertedIndexConfig":{"cleanupIntervalSeconds":60},"properties":[{"dataType":["int"],"name":"class_id"},{"dataType":["int"],"name":"***"},{"dataType":["int"],"name":"***"},{"dataType":["int"],"name":"***"},{"dataType":["int"],"name":"***"},{"dataType":["int"],"name":"***"},{"dataType":["int"],"name":"***"},{"dataType":["string"],"name":"***","tokenization":"word"}],"shardingConfig":{"virtualPerPhysical":128,"desiredCount":1,"actualCount":1,"desiredVirtualCount":128,"actualVirtualCount":128,"key":"_id","strategy":"hash","function":"murmur3"},"vectorIndexConfig":{"skip":false,"cleanupIntervalSeconds":300,"maxConnections":64,"efConstruction":128,"ef":-1,"dynamicEfMin":100,"dynamicEfMax":500,"dynamicEfFactor":8,"vectorCacheMaxObjects":2000000,"flatSearchCutoff":40000},"vectorIndexType":"hnsw","vectorizer":"none"},{"class":"***_***1_v1","description":"*** hu.","invertedIndexConfig":{"cleanupIntervalSeconds":60},"properties":[{"dataType":["int"],"name":"class_id"},{"dataType":["int"],"name":"***"},{"dataType":["int"],"name":"***"},{"dataType":["int"],"name":"***"},{"dataType":["int"],"name":"***"},{"dataType":["int"],"name":"***"},{"dataType":["int"],"name":"***"},{"dataType":["string"],"name":"***","tokenization":"word"}],"shardingConfig":{"virtualPerPhysical":128,"desiredCount":1,"actualCount":1,"desiredVirtualCount":128,"actualVirtualCount":128,"key":"_id","strategy":"hash","function":"murmur3"},"vectorIndexConfig":{"skip":false,"cleanupIntervalSeconds":300,"maxConnections":64,"efConstruction":128,"ef":-1,"dynamicEfMin":100,"dynamicEfMax":500,"dynamicEfFactor":8,"vectorCacheMaxObjects":2000000,"flatSearchCutoff":40000},"vectorIndexType":"hnsw","vectorizer":"none"},{"class":"***_***_v1","description":"*** combr.","invertedIndexConfig":{"cleanupIntervalSeconds":60},"properties":[{"dataType":["int"],"name":"class_id"},{"dataType":["int"],"name":"***"},{"dataType":["int"],"name":"***"},{"dataType":["int"],"name":"***"},{"dataType":["int"],"name":"***"},{"dataType":["int"],"name":"***"},{"dataType":["int"],"name":"***"},{"dataType":["string"],"name":"***","tokenization":"word"}],"shardingConfig":{"virtualPerPhysical":128,"desiredCount":1,"actualCount":1,"desiredVirtualCount":128,"actualVirtualCount":128,"key":"_id","strategy":"hash","function":"murmur3"},"vectorIndexConfig":{"skip":false,"cleanupIntervalSeconds":300,"maxConnections":64,"efConstruction":128,"ef":-1,"dynamicEfMin":100,"dynamicEfMax":500,"dynamicEfFactor":8,"vectorCacheMaxObjects":2000000,"flatSearchCutoff":40000},"vectorIndexType":"hnsw","vectorizer":"none"},{"class":"***_***_v1","description":"*** pt.","invertedIndexConfig":{"cleanupIntervalSeconds":60},"properties":[{"dataType":["int"],"name":"class_id"},{"dataType":["int"],"name":"***"},{"dataType":["int"],"name":"***"},{"dataType":["int"],"name":"***"},{"dataType":["int"],"name":"***"},{"dataType":["int"],"name":"***"},{"dataType":["int"],"name":"***"},{"dataType":["string"],"name":"***","tokenization":"word"}],"shardingConfig":{"virtualPerPhysical":128,"desiredCount":1,"actualCount":1,"desiredVirtualCount":128,"actualVirtualCount":128,"key":"_id","strategy":"hash","function":"murmur3"},"vectorIndexConfig":{"skip":false,"cleanupIntervalSeconds":300,"maxConnections":64,"efConstruction":128,"ef":-1,"dynamicEfMin":100,"dynamicEfMax":500,"dynamicEfFactor":8,"vectorCacheMaxObjects":2000000,"flatSearchCutoff":40000},"vectorIndexType":"hnsw","vectorizer":"none"},{"class":"***_***_v1","description":"*** nl.","invertedIndexConfig":{"cleanupIntervalSeconds":60},"properties":[{"dataType":["int"],"name":"class_id"},{"dataType":["int"],"name":"***"},{"dataType":["int"],"name":"***"},{"dataType":["int"],"name":"***"},{"dataType":["int"],"name":"***"},{"dataType":["int"],"name":"***"},{"dataType":["int"],"name":"***"},{"dataType":["string"],"name":"***","tokenization":"word"}],"shardingConfig":{"virtualPerPhysical":128,"desiredCount":1,"actualCount":1,"desiredVirtualCount":128,"actualVirtualCount":128,"key":"_id","strategy":"hash","function":"murmur3"},"vectorIndexConfig":{"skip":false,"cleanupIntervalSeconds":300,"maxConnections":64,"efConstruction":128,"ef":-1,"dynamicEfMin":100,"dynamicEfMax":500,"dynamicEfFactor":8,"vectorCacheMaxObjects":2000000,"flatSearchCutoff":40000},"vectorIndexType":"hnsw","vectorizer":"none"},{"class":"***_***_v1","description":"*** cz.","invertedIndexConfig":{"cleanupIntervalSeconds":60},"properties":[{"dataType":["int"],"name":"class_id"},{"dataType":["int"],"name":"***"},{"dataType":["int"],"name":"***"},{"dataType":["int"],"name":"***"},{"dataType":["int"],"name":"***"},{"dataType":["int"],"name":"***"},{"dataType":["int"],"name":"***"},{"dataType":["string"],"name":"***","tokenization":"word"}],"shardingConfig":{"virtualPerPhysical":128,"desiredCount":1,"actualCount":1,"desiredVirtualCount":128,"actualVirtualCount":128,"key":"_id","strategy":"hash","function":"murmur3"},"vectorIndexConfig":{"skip":false,"cleanupIntervalSeconds":300,"maxConnections":64,"efConstruction":128,"ef":-1,"dynamicEfMin":100,"dynamicEfMax":500,"dynamicEfFactor":8,"vectorCacheMaxObjects":2000000,"flatSearchCutoff":40000},"vectorIndexType":"hnsw","vectorizer":"none"},{"class":"***_***_v1","description":"*** cz.","invertedIndexConfig":{"cleanupIntervalSeconds":60},"properties":[{"dataType":["int"],"name":"class_id"},{"dataType":["int"],"name":"***"},{"dataType":["int"],"name":"***"},{"dataType":["int"],"name":"***"},{"dataType":["int"],"name":"***"},{"dataType":["int"],"name":"***"},{"dataType":["int"],"name":"***"},{"dataType":["string"],"name":"***","tokenization":"word"}],"shardingConfig":{"virtualPerPhysical":128,"desiredCount":1,"actualCount":1,"desiredVirtualCount":128,"actualVirtualCount":128,"key":"_id","strategy":"hash","function":"murmur3"},"vectorIndexConfig":{"skip":false,"cleanupIntervalSeconds":300,"maxConnections":64,"efConstruction":128,"ef":-1,"dynamicEfMin":100,"dynamicEfMax":500,"dynamicEfFactor":8,"vectorCacheMaxObjects":2000000,"flatSearchCutoff":40000},"vectorIndexType":"hnsw","vectorizer":"none"}]}},"time":"2022-04-06T11:06:49Z"}
{"action":"graphql_rebuild","level":"debug","msg":"successfully rebuild graphql schema","time":"2022-04-06T11:06:49Z"}
{"action":"restapi_management","level":"info","msg":"Serving weaviate at http://:8080","time":"2022-04-06T11:06:49Z"}
{"action":"hnsw_vector_cache_prefill_level","count":228051,"hnsw_level":0,"index_id":"***_***_v1_VFrq4mcLtG0m","level":"debug","msg":"prefilled level in vector cache","time":"2022-04-06T11:06:52Z","took":3531471315}
{"action":"hnsw_vector_cache_prefill","count":275000,"index_id":"***_***_v1_VFrq4mcLtG0m","level":"info","limit":2000000,"msg":"prefilled vector cache","time":"2022-04-06T11:06:52Z","took":3633373030}
{"action":"restapi_request","level":"debug","method":"POST","msg":"received HTTP request","time":"2022-04-06T11:07:05Z","url":{"Scheme":"","Opaque":"","User":null,"Host":"","Path":"/v1/graphql","RawPath":"","ForceQuery":false,"RawQuery":"","Fragment":"","RawFragment":""}}
{"action":"restapi_request","level":"debug","method":"POST","msg":"received HTTP request","time":"2022-04-06T11:07:09Z","url":{"Scheme":"","Opaque":"","User":null,"Host":"","Path":"/v1/graphql","RawPath":"","ForceQuery":false,"RawQuery":"","Fragment":"","RawFragment":""}}
{"action":"restapi_request","level":"debug","method":"POST","msg":"received HTTP request","time":"2022-04-06T11:07:11Z","url":{"Scheme":"","Opaque":"","User":null,"Host":"","Path":"/v1/graphql","RawPath":"","ForceQuery":false,"RawQuery":"","Fragment":"","RawFragment":""}}
{"action":"restapi_request","level":"debug","method":"POST","msg":"received HTTP request","time":"2022-04-06T11:07:12Z","url":{"Scheme":"","Opaque":"","User":null,"Host":"","Path":"/v1/graphql","RawPath":"","ForceQuery":false,"RawQuery":"","Fragment":"","RawFragment":""}}
{"action":"restapi_request","level":"debug","method":"POST","msg":"received HTTP request","time":"2022-04-06T11:07:13Z","url":{"Scheme":"","Opaque":"","User":null,"Host":"","Path":"/v1/graphql","RawPath":"","ForceQuery":false,"RawQuery":"","Fragment":"","RawFragment":""}}
{"action":"restapi_request","level":"debug","method":"POST","msg":"received HTTP request","time":"2022-04-06T11:07:14Z","url":{"Scheme":"","Opaque":"","User":null,"Host":"","Path":"/v1/graphql","RawPath":"","ForceQuery":false,"RawQuery":"","Fragment":"","RawFragment":""}}
{"action":"restapi_request","level":"debug","method":"POST","msg":"received HTTP request","time":"2022-04-06T11:07:15Z","url":{"Scheme":"","Opaque":"","User":null,"Host":"","Path":"/v1/graphql","RawPath":"","ForceQuery":false,"RawQuery":"","Fragment":"","RawFragment":""}}
{"action":"restapi_request","level":"debug","method":"POST","msg":"received HTTP request","time":"2022-04-06T11:07:15Z","url":{"Scheme":"","Opaque":"","User":null,"Host":"","Path":"/v1/graphql","RawPath":"","ForceQuery":false,"RawQuery":"","Fragment":"","RawFragment":""}}
{"action":"restapi_request","level":"debug","method":"POST","msg":"received HTTP request","time":"2022-04-06T11:07:16Z","url":{"Scheme":"","Opaque":"","User":null,"Host":"","Path":"/v1/graphql","RawPath":"","ForceQuery":false,"RawQuery":"","Fragment":"","RawFragment":""}}
{"action":"restapi_request","level":"debug","method":"POST","msg":"received HTTP request","time":"2022-04-06T11:07:17Z","url":{"Scheme":"","Opaque":"","User":null,"Host":"","Path":"/v1/graphql","RawPath":"","ForceQuery":false,"RawQuery":"","Fragment":"","RawFragment":""}}
{"action":"restapi_request","level":"debug","method":"POST","msg":"received HTTP request","time":"2022-04-06T11:07:18Z","url":{"Scheme":"","Opaque":"","User":null,"Host":"","Path":"/v1/graphql","RawPath":"","ForceQuery":false,"RawQuery":"","Fragment":"","RawFragment":""}}
{"action":"restapi_request","level":"debug","method":"POST","msg":"received HTTP request","time":"2022-04-06T11:07:18Z","url":{"Scheme":"","Opaque":"","User":null,"Host":"","Path":"/v1/graphql","RawPath":"","ForceQuery":false,"RawQuery":"","Fragment":"","RawFragment":""}}
{"action":"restapi_request","level":"debug","method":"POST","msg":"received HTTP request","time":"2022-04-06T11:07:19Z","url":{"Scheme":"","Opaque":"","User":null,"Host":"","Path":"/v1/graphql","RawPath":"","ForceQuery":false,"RawQuery":"","Fragment":"","RawFragment":""}}
{"action":"restapi_request","level":"debug","method":"POST","msg":"received HTTP request","time":"2022-04-06T11:07:19Z","url":{"Scheme":"","Opaque":"","User":null,"Host":"","Path":"/v1/graphql","RawPath":"","ForceQuery":false,"RawQuery":"","Fragment":"","RawFragment":""}}
{"action":"restapi_request","level":"debug","method":"POST","msg":"received HTTP request","time":"2022-04-06T11:07:20Z","url":{"Scheme":"","Opaque":"","User":null,"Host":"","Path":"/v1/graphql","RawPath":"","ForceQuery":false,"RawQuery":"","Fragment":"","RawFragment":""}}
{"action":"restapi_request","level":"debug","method":"POST","msg":"received HTTP request","time":"2022-04-06T11:07:21Z","url":{"Scheme":"","Opaque":"","User":null,"Host":"","Path":"/v1/graphql","RawPath":"","ForceQuery":false,"RawQuery":"","Fragment":"","RawFragment":""}}
{"action":"restapi_request","level":"debug","method":"POST","msg":"received HTTP request","time":"2022-04-06T11:07:21Z","url":{"Scheme":"","Opaque":"","User":null,"Host":"","Path":"/v1/graphql","RawPath":"","ForceQuery":false,"RawQuery":"","Fragment":"","RawFragment":""}}
{"action":"restapi_request","level":"debug","method":"POST","msg":"received HTTP request","time":"2022-04-06T11:07:22Z","url":{"Scheme":"","Opaque":"","User":null,"Host":"","Path":"/v1/graphql","RawPath":"","ForceQuery":false,"RawQuery":"","Fragment":"","RawFragment":""}}
{"action":"restapi_request","level":"debug","method":"POST","msg":"received HTTP request","time":"2022-04-06T11:07:23Z","url":{"Scheme":"","Opaque":"","User":null,"Host":"","Path":"/v1/graphql","RawPath":"","ForceQuery":false,"RawQuery":"","Fragment":"","RawFragment":""}}
{"action":"restapi_request","level":"debug","method":"POST","msg":"received HTTP request","time":"2022-04-06T11:07:23Z","url":{"Scheme":"","Opaque":"","User":null,"Host":"","Path":"/v1/graphql","RawPath":"","ForceQuery":false,"RawQuery":"","Fragment":"","RawFragment":""}}
{"action":"restapi_request","level":"debug","method":"POST","msg":"received HTTP request","time":"2022-04-06T11:07:24Z","url":{"Scheme":"","Opaque":"","User":null,"Host":"","Path":"/v1/graphql","RawPath":"","ForceQuery":false,"RawQuery":"","Fragment":"","RawFragment":""}}
{"action":"restapi_request","level":"debug","method":"POST","msg":"received HTTP request","time":"2022-04-06T11:07:25Z","url":{"Scheme":"","Opaque":"","User":null,"Host":"","Path":"/v1/graphql","RawPath":"","ForceQuery":false,"RawQuery":"","Fragment":"","RawFragment":""}}
{"action":"restapi_request","level":"debug","method":"POST","msg":"received HTTP request","time":"2022-04-06T11:07:26Z","url":{"Scheme":"","Opaque":"","User":null,"Host":"","Path":"/v1/graphql","RawPath":"","ForceQuery":false,"RawQuery":"","Fragment":"","RawFragment":""}}
{"action":"restapi_request","level":"debug","method":"POST","msg":"received HTTP request","time":"2022-04-06T11:07:27Z","url":{"Scheme":"","Opaque":"","User":null,"Host":"","Path":"/v1/graphql","RawPath":"","ForceQuery":false,"RawQuery":"","Fragment":"","RawFragment":""}}
{"action":"restapi_request","level":"debug","method":"POST","msg":"received HTTP request","time":"2022-04-06T11:07:27Z","url":{"Scheme":"","Opaque":"","User":null,"Host":"","Path":"/v1/graphql","RawPath":"","ForceQuery":false,"RawQuery":"","Fragment":"","RawFragment":""}}
{"action":"restapi_request","level":"debug","method":"POST","msg":"received HTTP request","time":"2022-04-06T11:07:28Z","url":{"Scheme":"","Opaque":"","User":null,"Host":"","Path":"/v1/graphql","RawPath":"","ForceQuery":false,"RawQuery":"","Fragment":"","RawFragment":""}}
{"action":"restapi_request","level":"debug","method":"POST","msg":"received HTTP request","time":"2022-04-06T11:07:29Z","url":{"Scheme":"","Opaque":"","User":null,"Host":"","Path":"/v1/graphql","RawPath":"","ForceQuery":false,"RawQuery":"","Fragment":"","RawFragment":""}}
{"action":"restapi_request","level":"debug","method":"POST","msg":"received HTTP request","time":"2022-04-06T11:07:30Z","url":{"Scheme":"","Opaque":"","User":null,"Host":"","Path":"/v1/graphql","RawPath":"","ForceQuery":false,"RawQuery":"","Fragment":"","RawFragment":""}}
{"action":"restapi_request","level":"debug","method":"POST","msg":"received HTTP request","time":"2022-04-06T11:07:30Z","url":{"Scheme":"","Opaque":"","User":null,"Host":"","Path":"/v1/graphql","RawPath":"","ForceQuery":false,"RawQuery":"","Fragment":"","RawFragment":""}}
{"action":"restapi_request","level":"debug","method":"POST","msg":"received HTTP request","time":"2022-04-06T11:07:31Z","url":{"Scheme":"","Opaque":"","User":null,"Host":"","Path":"/v1/graphql","RawPath":"","ForceQuery":false,"RawQuery":"","Fragment":"","RawFragment":""}}
{"action":"restapi_request","level":"debug","method":"POST","msg":"received HTTP request","time":"2022-04-06T11:07:32Z","url":{"Scheme":"","Opaque":"","User":null,"Host":"","Path":"/v1/graphql","RawPath":"","ForceQuery":false,"RawQuery":"","Fragment":"","RawFragment":""}}
{"action":"restapi_request","level":"debug","method":"POST","msg":"received HTTP request","time":"2022-04-06T11:07:33Z","url":{"Scheme":"","Opaque":"","User":null,"Host":"","Path":"/v1/graphql","RawPath":"","ForceQuery":false,"RawQuery":"","Fragment":"","RawFragment":""}}
{"action":"restapi_request","level":"debug","method":"POST","msg":"received HTTP request","time":"2022-04-06T11:07:33Z","url":{"Scheme":"","Opaque":"","User":null,"Host":"","Path":"/v1/graphql","RawPath":"","ForceQuery":false,"RawQuery":"","Fragment":"","RawFragment":""}}
{"action":"restapi_request","level":"debug","method":"POST","msg":"received HTTP request","time":"2022-04-06T11:07:34Z","url":{"Scheme":"","Opaque":"","User":null,"Host":"","Path":"/v1/graphql","RawPath":"","ForceQuery":false,"RawQuery":"","Fragment":"","RawFragment":""}}
{"action":"restapi_request","level":"debug","method":"POST","msg":"received HTTP request","time":"2022-04-06T11:07:35Z","url":{"Scheme":"","Opaque":"","User":null,"Host":"","Path":"/v1/graphql","RawPath":"","ForceQuery":false,"RawQuery":"","Fragment":"","RawFragment":""}}
{"action":"restapi_request","level":"debug","method":"POST","msg":"received HTTP request","time":"2022-04-06T11:07:35Z","url":{"Scheme":"","Opaque":"","User":null,"Host":"","Path":"/v1/graphql","RawPath":"","ForceQuery":false,"RawQuery":"","Fragment":"","RawFragment":""}}
{"action":"restapi_request","level":"debug","method":"POST","msg":"received HTTP request","time":"2022-04-06T11:07:36Z","url":{"Scheme":"","Opaque":"","User":null,"Host":"","Path":"/v1/graphql","RawPath":"","ForceQuery":false,"RawQuery":"","Fragment":"","RawFragment":""}}
{"action":"restapi_request","level":"debug","method":"POST","msg":"received HTTP request","time":"2022-04-06T11:07:37Z","url":{"Scheme":"","Opaque":"","User":null,"Host":"","Path":"/v1/graphql","RawPath":"","ForceQuery":false,"RawQuery":"","Fragment":"","RawFragment":""}}
{"action":"restapi_request","level":"debug","method":"POST","msg":"received HTTP request","time":"2022-04-06T11:07:38Z","url":{"Scheme":"","Opaque":"","User":null,"Host":"","Path":"/v1/graphql","RawPath":"","ForceQuery":false,"RawQuery":"","Fragment":"","RawFragment":""}}
{"action":"restapi_request","level":"debug","method":"POST","msg":"received HTTP request","time":"2022-04-06T11:07:38Z","url":{"Scheme":"","Opaque":"","User":null,"Host":"","Path":"/v1/graphql","RawPath":"","ForceQuery":false,"RawQuery":"","Fragment":"","RawFragment":""}}
{"action":"restapi_request","level":"debug","method":"POST","msg":"received HTTP request","time":"2022-04-06T11:07:39Z","url":{"Scheme":"","Opaque":"","User":null,"Host":"","Path":"/v1/graphql","RawPath":"","ForceQuery":false,"RawQuery":"","Fragment":"","RawFragment":""}}
{"action":"restapi_request","level":"debug","method":"POST","msg":"received HTTP request","time":"2022-04-06T11:07:39Z","url":{"Scheme":"","Opaque":"","User":null,"Host":"","Path":"/v1/graphql","RawPath":"","ForceQuery":false,"RawQuery":"","Fragment":"","RawFragment":""}}
{"action":"restapi_request","level":"debug","method":"POST","msg":"received HTTP request","time":"2022-04-06T11:07:40Z","url":{"Scheme":"","Opaque":"","User":null,"Host":"","Path":"/v1/graphql","RawPath":"","ForceQuery":false,"RawQuery":"","Fragment":"","RawFragment":""}}
{"action":"restapi_request","level":"debug","method":"POST","msg":"received HTTP request","time":"2022-04-06T11:07:41Z","url":{"Scheme":"","Opaque":"","User":null,"Host":"","Path":"/v1/graphql","RawPath":"","ForceQuery":false,"RawQuery":"","Fragment":"","RawFragment":""}}
{"action":"restapi_request","level":"debug","method":"POST","msg":"received HTTP request","time":"2022-04-06T11:07:41Z","url":{"Scheme":"","Opaque":"","User":null,"Host":"","Path":"/v1/graphql","RawPath":"","ForceQuery":false,"RawQuery":"","Fragment":"","RawFragment":""}}
{"action":"restapi_request","level":"debug","method":"POST","msg":"received HTTP request","time":"2022-04-06T11:07:42Z","url":{"Scheme":"","Opaque":"","User":null,"Host":"","Path":"/v1/graphql","RawPath":"","ForceQuery":false,"RawQuery":"","Fragment":"","RawFragment":""}}
{"action":"restapi_request","level":"debug","method":"POST","msg":"received HTTP request","time":"2022-04-06T11:07:43Z","url":{"Scheme":"","Opaque":"","User":null,"Host":"","Path":"/v1/graphql","RawPath":"","ForceQuery":false,"RawQuery":"","Fragment":"","RawFragment":""}}
{"action":"restapi_request","level":"debug","method":"POST","msg":"received HTTP request","time":"2022-04-06T11:07:43Z","url":{"Scheme":"","Opaque":"","User":null,"Host":"","Path":"/v1/graphql","RawPath":"","ForceQuery":false,"RawQuery":"","Fragment":"","RawFragment":""}}
{"action":"restapi_request","level":"debug","method":"POST","msg":"received HTTP request","time":"2022-04-06T11:07:44Z","url":{"Scheme":"","Opaque":"","User":null,"Host":"","Path":"/v1/graphql","RawPath":"","ForceQuery":false,"RawQuery":"","Fragment":"","RawFragment":""}}
{"action":"restapi_request","level":"debug","method":"POST","msg":"received HTTP request","time":"2022-04-06T11:07:44Z","url":{"Scheme":"","Opaque":"","User":null,"Host":"","Path":"/v1/graphql","RawPath":"","ForceQuery":false,"RawQuery":"","Fragment":"","RawFragment":""}}
{"action":"restapi_request","level":"debug","method":"POST","msg":"received HTTP request","time":"2022-04-06T11:07:45Z","url":{"Scheme":"","Opaque":"","User":null,"Host":"","Path":"/v1/graphql","RawPath":"","ForceQuery":false,"RawQuery":"","Fragment":"","RawFragment":""}}
{"action":"restapi_request","level":"debug","method":"POST","msg":"received HTTP request","time":"2022-04-06T11:07:46Z","url":{"Scheme":"","Opaque":"","User":null,"Host":"","Path":"/v1/graphql","RawPath":"","ForceQuery":false,"RawQuery":"","Fragment":"","RawFragment":""}}
panic: runtime error: slice bounds out of range [:226] with capacity 128

goroutine 569 [running]:
github.com/semi-technologies/weaviate/adapters/repos/db/vector/hnsw.(*connList).Get(...)
    /go/src/github.com/semi-technologies/weaviate/adapters/repos/db/vector/hnsw/pools.go:149
github.com/semi-technologies/weaviate/adapters/repos/db/vector/hnsw.(*hnsw).searchLayerByVector(0xc0004baf00, 0xc74424a800, 0x600, 0x600, 0xc71620d340, 0x80, 0x0, 0x0, 0x0, 0x0, ...)
    /go/src/github.com/semi-technologies/weaviate/adapters/repos/db/vector/hnsw/search.go:117 +0xbd2
github.com/semi-technologies/weaviate/adapters/repos/db/vector/hnsw.(*neighborFinderConnector).doAtLevel(0xc7bdf6bba0, 0x0, 0xc74424a800, 0x600)
    /go/src/github.com/semi-technologies/weaviate/adapters/repos/db/vector/hnsw/neighbor_connections.go:85 +0xf5
github.com/semi-technologies/weaviate/adapters/repos/db/vector/hnsw.(*neighborFinderConnector).Do(0xc7bdf6bba0, 0x0, 0x600000000000000)
    /go/src/github.com/semi-technologies/weaviate/adapters/repos/db/vector/hnsw/neighbor_connections.go:68 +0xbc
github.com/semi-technologies/weaviate/adapters/repos/db/vector/hnsw.(*hnsw).findAndConnectNeighbors(0xc0004baf00, 0xc026694000, 0x21f235, 0xc74424a800, 0x600, 0x600, 0x0, 0x3, 0xc301fbc120, 0x0, ...)
    /go/src/github.com/semi-technologies/weaviate/adapters/repos/db/vector/hnsw/neighbor_connections.go:26 +0xd3
github.com/semi-technologies/weaviate/adapters/repos/db/vector/hnsw.(*hnsw).reassignNeighborsOf(0xc0004baf00, 0xc301fbc120, 0xc048735380, 0x0)
    /go/src/github.com/semi-technologies/weaviate/adapters/repos/db/vector/hnsw/delete.go:247 +0x4cf
github.com/semi-technologies/weaviate/adapters/repos/db/vector/hnsw.(*hnsw).CleanUpTombstonedNodes(0xc0004baf00, 0xc0488f5620, 0x0)
    /go/src/github.com/semi-technologies/weaviate/adapters/repos/db/vector/hnsw/delete.go:124 +0x75
github.com/semi-technologies/weaviate/adapters/repos/db/vector/hnsw.(*hnsw).registerTombstoneCleanup.func1(0xc0004baf00)
    /go/src/github.com/semi-technologies/weaviate/adapters/repos/db/vector/hnsw/startup.go:134 +0xda
created by github.com/semi-technologies/weaviate/adapters/repos/db/vector/hnsw.(*hnsw).registerTombstoneCleanup
    /go/src/github.com/semi-technologies/weaviate/adapters/repos/db/vector/hnsw/startup.go:127 +0x53
Stream closed EOF for weaviate/weaviate-statefulset-0 (weaviate)
```

## Schema

```json
{
	"class": "***_v1",
	"description": "***",
	"invertedIndexConfig": {
		"cleanupIntervalSeconds": 60
	},
	"properties": [
		{
			"dataType": [
				"int"
			],
			"name": "***_id"
		},
		{
			"dataType": [
				"int"
			],
			"name": "***_id"
		},
		{
			"dataType": [
				"int"
			],
			"name": "***_id"
		},
		{
			"dataType": [
				"int"
			],
			"name": "***_id"
		},
		{
			"dataType": [
				"int"
			],
			"name": "***_id"
		},
		{
			"dataType": [
				"int"
			],
			"name": "***_id"
		},
		{
			"dataType": [
				"string"
			],
			"name": "***"
		}
	],
	"shardingConfig": {
		"virtualPerPhysical": 128,
		"desiredCount": 1,
		"actualCount": 1,
		"desiredVirtualCount": 128,
		"actualVirtualCount": 128,
		"key": "_id",
		"strategy": "hash",
		"function": "murmur3"
	},
	"vectorIndexConfig": {
		"skip": false,
		"cleanupIntervalSeconds": 300,
		"maxConnections": 64,
		"efConstruction": 128,
		"ef": -1,
		"dynamicEfMin": 100,
		"dynamicEfMax": 500,
		"dynamicEfFactor": 8,
		"vectorCacheMaxObjects": 2000000,
		"flatSearchCutoff": 40000
	},
	"vectorIndexType": "hnsw",
	"vectorizer": "none"
}
```

## Configuration

```
AUTHENTICATION_ANONYMOUS_ACCESS_ENABLED: "true"
PERSISTENCE_DATA_PATH: "/var/lib/weaviate"
DEFAULT_VECTORIZER_MODULE: "none"
ENABLE_MODULES: ""
LOG_LEVEL: "debug"
AUTOSCHEMA_ENABLED: "false"
```

## System info

* EC2 instance: `r5b.2xlarge` with 128 GiB EBS disk
* Environment: Kubernetes and standard Docker image from https://registry.hub.docker.com/r/semitechnologies/weaviate
* Size of `/var/lib/weaviate` is ~40 GiB

--- a/adapters/repos/db/vector/hnsw/search.go
+++ b/adapters/repos/db/vector/hnsw/search.go
@@ -114,8 +114,28 @@ func (h *hnsw) searchLayerByVector(queryVector []float32,
 		}
 
 		candidateNode.Lock()
-		connections := h.pools.connList.Get(len(candidateNode.connections[level]))
-		defer h.pools.connList.Put(connections)
+
+		var connections *[]uint64
+
+		if len(candidateNode.connections[level]) > h.maximumConnectionsLayerZero {
+			// How is it possible that we could ever have more connections than the
+			// allowed maximum? It is not anymore, but there was a bug that allowed
+			// this to happen in versions prior to v1.12.0:
+			// https://github.com/semi-technologies/weaviate/issues/1868
+			//
+			// As a result the length of this slice is entirely unpredictable and we
+			// can no longer retrieve it from the pool. Instead we need to fallback
+			// to allocating a new slice.
+			//
+			// This was discovered as part of
+			// https://github.com/semi-technologies/weaviate/issues/1897
+			c := make([]uint64, len(candidateNode.connections[level]))
+			connections = &c
+		} else {
+			connections = h.pools.connList.Get(len(candidateNode.connections[level]))
+			defer h.pools.connList.Put(connections)
+		}
+
 		for i, conn := range candidateNode.connections[level] {
 			(*connections)[i] = conn
 		}

